string name
trajectory_msgs/JointTrajectory joint_trajectory